mail_send_error_activity/models/mail_mail.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) Cetmix OU <http://www.cetmix.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class MailMail(models.Model):
+    _inherit = "mail.mail"
+
+    def _prepare_mail_error_activity_vals(self):
+        """
+        Prepare mail error activity values
+        :return: dict of values
+        """
+        user_ids = self.author_id.user_ids
```

Comment:
it's not ids it's recordset, we need use "users" variable name

Reply:
Resolved